src/components/Sidebar/ActionNodeEditor.tsx

```diff
@@ -56,31 +57,31 @@ const ActionNodeEditor: FC<ActionNodeEditorProps> = ({ changeNodeType }) => {
 
   const changeCode = useCallback(
     (event: ChangeEvent<{ value: string }>): void => {
-      const currentNode = currentNodeRef.current as ActionNode;
+      const currentNode = currentNodeStatic as ActionNode;
```

Comment:
Would it be easier to just define currentNodeStatic as ActionNode instead of doing this in every callback?

Reply:
ya, good point